### PR TITLE
refactor: ➖ remove `chalk` as dependency

### PR DESCRIPTION
**Pull request by kopach**
fixes: #43

Some docs: https://stackoverflow.com/questions/4842424/list-of-ansi-color-escape-sequences

Tested on Linux & Mac
Haven't tested on **Windows**

## Simple test
image

### CODE REVIEW

**Comment by kopach** [comment]
@ValentinH , please review

**Comment by ValentinH** [comment]
The tests are pretty trustworthy. It seems that only the cases that check the actual behavior of tests failed by failOnConsole are failing.
Maybe the original chalk has a different behavior when used in tests/Jest?

**Comment by kopach** [comment]
as those test cases meant to fail, I guess, `toThrow`  https://jestjs.io/docs/expect#tothrowerror should be used instead. what do you think?

**Comment by ValentinH** [comment]
Unfortunately this isn't that easy. These tests are actually using Jest to test a Jest plugin. That's why we are using this "fixture" pattern.

**Comment by ValentinH** [comment]
Unfortunately this isn't that easy. These tests are actually using Jest to test a Jest plugin. That's why we are using this "fixture" pattern.

**Comment by kopach** [comment]
ok, found issue, will try to debug

File: index.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 const util = require('util')
-const chalk = require('chalk')
+
+const chalk = {
+  red: (str) => `\u001b[31m${str}\u001b[0m`,
+  gray: (str) => `\u001b[90m${str}\u001b[0m`,
+  white: (str) => `\u001b[37m${str}\u001b[0m`,
+  bold: (str) => `\u001b[1m${str}\u001b[22m`
```
Comment (kopach) [inline index.js:7]: tests in `chalk` repo which confirms current implementation is correct https://github.com/chalk/chalk/blob/f7b29ae8ef4fd2048e08aa361778d290ed10ce7a/test/chalk.js#L23C7-L23C7

File: index.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+}
 
 const defaultErrorMessage = (methodName, bold) =>
-  `Expected test not to call ${bold(`console.${methodName}()`)}.\n\n` +
+  `Expected test not to call ${(`console.${methodName}()`)}.\n\n` +
```
[the remark this answers is not in the document]
Comment (ValentinH) [reply index.js]: I've reverted this and slightly adjusted the test to be more flexible on the string assertion.

File: index.js (hunk 1 above)
Comment (ValentinH) [reply index.js]: Thanks, after looking at it, I've changed the closing sequence to match what was used by Chalk. (from 0m to 39m)

**Comment by ValentinH** [comment]
Thanks for this contribution. It was released as https://github.com/ValentinH/jest-fail-on-console/releases/tag/v3.1.2

**Pull request merged**